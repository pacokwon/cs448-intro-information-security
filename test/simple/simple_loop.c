#include <stdio.h>

int main()
{
    char input[65536];
    fgets(input, sizeof(input), stdin);

    for(int i = 0; i < 100; i++)
    {
        int x = 100 / i;
    }

    return 0;
}
