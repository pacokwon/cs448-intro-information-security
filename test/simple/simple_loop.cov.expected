3,9
3,22
3,5
